crates/shielder_bindings/Cargo.toml

```diff
@@ -43,6 +43,7 @@ multithreading-wasm = [
     "wasm-bindgen-rayon",
     "shielder-circuits/multithreading",
 ]
+circuits = []
```

Comment:
I was wondering if it makes sense to put some comments here, like what feature means. E.g. comment for this one would be `# Build wasm bindings without circuits` etc